apps/src/templates/projects/ProjectsList.jsx

```diff
@@ -133,3 +136,6 @@ const ProjectsList = React.createClass({
   }
 });
 export default ProjectsList;
+
+window.dashboard = window.dashboard || {};
+window.dashboard.ProjectsList = ProjectsList;
```

Comment:
Would be better if this happened in the entry file (apps/src/sites/studio/pages/projectsList.js) instead of here.